tests/test_functional.py

```diff
@@ -725,37 +726,37 @@ def test_http11_generator(self):
         self.assertRaises(ConnectionClosed, read_http, fp)
 
     def test_http11_list(self):
-        body = string.ascii_letters
-        to_send = "GET /list HTTP/1.1\r\nContent-Length: %d\r\n\r\n" % len(body)
+        body = bytes(string.ascii_letters, "latin-1")
```

Comment:
is this different from `string.ascii_letters.encode('latin-1')`? I'm not used to seeing this syntax.

Reply:
It is the same afaik.